envs/example/ci-full/group_vars/all.yml

```diff
@@ -1,6 +1,19 @@
 ---
 stack_env: example_ci-full
 ursula_os: 'ubuntu'
+undercloud_cidr:
+  - cidr: 192.168.0.0/22
+
+ceph:
```

Comment:
please move below 3 varibale blocks down near cinder block. We must also keep our envs readbale.